core/src/toga/widgets/webview.py

```diff
@@ -74,6 +74,13 @@ def url(self):
 
     @url.setter
     def url(self, value):
+        """Set the current URL of the web view.
+
+        Args:
+            value (str): The URL as a ``str``
+
+        Returns:
```

Comment:
We've been inconsistent about this in the past, but we're now trying to converge on Sphinx docstring format rather than Google/Napoleon format:
```suggestion
        :param value: The URL to display
```